File: EJERCICIO_3/EJERCICIOSJAVA/EJERCICIOSJAVA/SecondLaw.java
package EJERCICIOSJAVA;
class SecondLaw extends NewtonsLaw {
    private double force;
    private double mass;
    private double acceleration;

    public SecondLaw(double force, double mass, double acceleration) {
        super("Second Newton's Law", "Newton's second law states that the force applied to an object is directly proportional to the acceleration the object acquires.");
        this.force = force;
        this.mass = mass;
        this.acceleration = acceleration;
    }

    public double getForce() {
        return this.force;
    }

    public double getMass() {
        return this.mass;
    }

    public double getAcceleration() {
        return this.acceleration;
    }

    public double calculateLaw() {
        return this.force = this.mass * this.acceleration;
    }
}

File: EJERCICIO_3/EJERCICIOSPHP/secondlaw.php
<?php
class Secondlaw extends NewtonsLaw{
    private $force;
    private $bulk;
    private $acceleration;

    private $calculate;

    public function __construct( & $force, $bulk, $acceleration){
        parent ::__construct("Second Newton´s law", "Newton's second law states that the force applied to an object is directly proportional to the acceleration the object adquires." );
        $this->force =$force;
        $this->bulk = $bulk;
        $this->acceleration = $acceleration;
    }

    public function get_force(){ 
        return $this->force;
    }

    public function get_bulk(){ 
        return $this->bulk;
    }

    public function get_acceleration(){
        return $this->acceleration;
    }

    public function calculateLaw(){
        return $this->force = $this->bulk * $this->acceleration; 
    }

}
?>